### ISSUE TITLE
A11y_AzureDataStudio_Newquery_LabelsorInstructions: Visual label is not provided for "Select database" dropdown.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22675]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG3.3.2; #Linux; #Closed; #Regressed:05-15-23;

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to "File" and hit ENTER key. TAB to "New Query" and hit ENTER key.
4. TAB to "Select database" and observe that Visual label is not provided for "Select database" dropdown.

### Actual Result:​
Visual label is not provided for "Select database" dropdown.

### Expected Result:​
Appropriate visual label should be provided for "Select database" dropdown.

### User Impact:
Screen reader users will not the purpose of the dropdown and will not get information about it.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:
![Uploading Github label.png…]()

**Comment by PoojaNamde** [comment]
Primary Bug: https://github.com/microsoft/azuredatastudio-mysql/issues/178

**Comment by alanrenmsft** [comment]
@VijayaBharathyA Please use the latest insider build to verify the fix.

**Comment by PoojaNamde** [comment]
Verified this issue in the latest Azure data studio- Insider (Version: 1.45.0 Insider).
Issue is fixed, hence closing the bug.
